projects/ngx-openlayers/src/lib/sources/raster.component.ts

```diff
@@ -25,7 +25,7 @@ export class SourceRasterComponent extends SourceComponent implements AfterConte
   @Input()
   threads?: number;
   @Input()
-  lib?: Object;
+  lib?: any;
```

Comment:
Can't we also have `{ [key: string]: any }` here?

Reply:
According to the [doc of openlayers](https://openlayers.org/en/latest/apidoc/module-ol_source_Raster-RasterSource.html) lib is an object.

It is documented in js-doc like this:

`@property {Object} [lib] Functions that will be made available to operations run in a worker.`

And that becomes in Typescript [@type/ol](
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/ol/source/Raster.d.ts):
`lib?: any;`